jobs/examples/ex1_frameworked_job.py

```diff
@@ -1,20 +1,24 @@
 """Same as ex1_full_sql_job.sql but allows access to spark for more complex ops (not used here but in ex2_frameworked_job.py)."""
 from yaetos.etl_utils import ETL_Base, Commandliner
+from pyspark import sql
 
 
 class Job(ETL_Base):
-    def transform(self, some_events, other_events):
-        df = self.query("""
+    def transform(
+        self, some_events="some_events", other_events="other_events"
+    ) -> sql.DataFrame:
+        return self.query(
+            f"""
             SELECT se.session_id, count(*) as count_events
-            FROM some_events se
-            JOIN other_events oe on se.session_id=oe.session_id
+            FROM {some_events} se
+            JOIN {other_events} oe on se.session_id=oe.session_id
```

Comment:
I like the idea. I have been thinking of doing something like that but haven't gotten to it yet. I think it would need to be done differently since the input variables are not expected to be strings.